File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    n = len(commands)
    
    num = 0
    for j in range(n):
        srt , end, target = commands[j]
        # print(srt , end, target)
        temp = sorted(array[srt-1: end])
        answer.append(temp[target-1])
    
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int[] solution(int[] array, int[,] commands) {
        int n = commands.GetLength(0);
        // Console.WriteLine(n);
        int[] answer = new int[n];
        
        for(int i=0; i<n; i++){
            int start = commands[i, 0] - 1;
            int end = commands[i, 1] - 1;
            int target = commands[i, 2] -1;
            int len = end - start + 1;
            int[] arr = new int[len];
            
            Array.Copy(array, start, arr, 0, len);
            Array.Sort(arr);
            answer[i] = arr[target];
            
        }
        
        return answer;
    }
}